_posts/2019-07-25-testy-frontendu-okiem-full-stacka.md

```diff
@@ -0,0 +1,136 @@
+---
+layout:    post
+title:     "Testy jednostkowe frontendu okiem programisty full stack"
+published: true
+date:      2019-07-25 10:00:00 +0100
+author:    mmendlik
+tags:
+    - unit test
+    - jasmine
+    - angular
+---
+
+Niełatwo znaleźć wymówkę, żeby nie pisać testów jednostkowych. Obecność frameworków ułatwiających tę czynność w projektach, z którymi stykamy się na co dzień, nie powinna na żadnym chociaż trochę doświadczonym programiście robić wrażenia. Nie trzeba pewnie Cię przekonywać, że **jedne z wielu zalet pisania testów jednostkowych**, to:
```

Comment:
> Nie trzeba pewnie Cię przekonywać

Bezpośredni zwrot do czytelnika to fajny zabieg stylistyczny, ale spodziewałbym się, że cały art jest napisany w takiej konwencji

Reply:
Rozumiem, sparafrazuję to jakoś